flink-streaming-java/src/main/java/org/apache/flink/streaming/api/graph/StreamNode.java

```diff
@@ -84,7 +84,7 @@
 
 	private String transformationUID;
 	private String userHash;
-	private boolean sortedInputs = false;
+	private Map<Integer, StreamConfig.InputRequirement> inputRequirements = new HashMap<>();
```

Comment:
Why not making this `final`?

Also nit-pick: I find it is nicer to initialize fields in the constructor but feel free to ignore this.

Reply:
None of the others are initialized in the constructor that's why I went with this one. But yes, I would agree!